projects/novo-elements/src/elements/form/FieldInteractionApi.ts

```diff
@@ -415,6 +415,18 @@ export class FieldInteractionApi {
     return this.modalService.open(ControlPromptModal, { changes: changes, key: key }).onClosed;
   }
 
+  public promptCustomModal(key: string,
+    modalConfig: {
```

Comment:
Please add a demo to the field interactions section here: http://bullhorn.github.io/novo-elements/#/utils/field-interactions

Also, please include screenshots to the demo in the PR description.